test/params.js

```diff
@@ -25,6 +25,22 @@ describe('LDNODE params', function () {
         .expect(200, done)
     })
 
+    it('should return local network requests', function (done) {
```

Comment:
Isn't this description misleading? The test exists to make sure that the server does **not** serve anything that is on the local network.